shared/selene/data/tagging/__init__.py

```diff
@@ -0,0 +1,23 @@
+# Mycroft Server - Backend
+# Copyright (C) 2020 Mycroft AI Inc
+# SPDX-License-Identifier: 	AGPL-3.0-or-later
+#
+# This file is part of the Mycroft Server.
+#
+# The Mycroft Server is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+from .entity.file import WakeWordFile
+from .entity.file_location import TaggingFileLocation
```

Comment:
Just curious, why does a tagging entity need a file location?

Reply:
Is the naming what is confusing?  I just prefixed the word "tagging" because it was in the tagging schema.  I could just call it "FileLocation"